code/game/gamemodes/colonialmarines/colonialmarines.dm

```diff
@@ -81,13 +81,13 @@
 /datum/game_mode/colonialmarines/proc/map_announce()
 	switch(SSmapping.config.map_name)
 		if(MAP_LV_624)
-			command_announcement.Announce("An automated distress signal has been received from archaeology site Lazarus Landing, on border world LV-624. A response team from the [MAIN_SHIP_NAME] will be dispatched shortly to investigate.", "[MAIN_SHIP_NAME]")
+			command_announcement.Announce("A faint distress signal has been picked up by our scanners, which have tracked the source to a third generation colony, known as LV-624. Through use of bluespace drive tech, the [MAIN_SHIP_NAME] has jumped within range of the colony. TGMC, gear up and get ready to respond!", "[MAIN_SHIP_NAME]")
 		if(MAP_ICE_COLONY)
-			command_announcement.Announce("An automated distress signal has been received from archaeology site \"Shiva's Snowball\", on border ice world \"Ifrit\". A response team from the [MAIN_SHIP_NAME] will be dispatched shortly to investigate.", "[MAIN_SHIP_NAME]")
+			command_announcement.Announce("A garbled, unintelligible communications message was broadcasted over a general frequency, and picked up by our comms relay. The message appears to have come from a second generation settlement, located on an ice cold planet. The [MAIN_SHIP_NAME] is moving into the sector with thrusters at max throttle. TGMC, get briefed and then move out! ", "[MAIN_SHIP_NAME]")
 		if(MAP_BIG_RED)
-			command_announcement.Announce("We've lost contact with the Nanotrasen's research facility, [SSmapping.config.map_name]. The [MAIN_SHIP_NAME] has been dispatched to assist.", "[MAIN_SHIP_NAME]")
+			command_announcement.Announce("A second generation colony, [SSmapping.config.map_name], has had a beacon transmitting the same signal, nonstop. Attempts to hail the colony over comms have proved futile. Because the [MAIN_SHIP_NAME] was at a nearby drydock, it has been dispatched to figure out what's wrong. TGMC, prepare to deploy! ", "[MAIN_SHIP_NAME]")
```

Comment:
It would be perhaps better to remove the embedded map name expression as the code name for Big Red is Big Red, but elsewhere it's named Solaris Ridge.

Reply:
Change has been made.